tests/cypress/tests-e2e/menuTray.cy.js

```diff
@@ -213,57 +213,117 @@ function checkScrollbarVisibility(topicsScrollable, activeLayersScrollable) {
+            checkOpenSections([])
+            measureMenu(false)
 
-        cy.get(menuActiveLayersHeaderSelector).click()
-        waitForAnimationsToFinish()
-        checkOpenSections(['activeLayers'])
-        measureMenu(true)
+            cy.get(menuActiveLayersHeaderSelector).click()
+            waitForAnimationsToFinish()
+            checkOpenSections(['activeLayers'])
+            measureMenu(true)
 
-        cy.get(menuHelpSectionHeader).click()
-        waitForAnimationsToFinish()
-        checkOpenSections(['help'])
-        measureMenu(false)
+            cy.get(menuHelpSectionHeader).click()
+            waitForAnimationsToFinish()
+            checkOpenSections(['help'])
+            measureMenu(false)
 
-        cy.get(menuShareHeaderSelector).click()
-        waitForAnimationsToFinish()
-        checkOpenSections(['share'])
-        measureMenu(false)
+            cy.get(menuShareHeaderSelector).click()
+            waitForAnimationsToFinish()
+            checkOpenSections(['share'])
+            measureMenu(false)
 
-        cy.get(menuTopicHeaderSelector).click()
-        waitForAnimationsToFinish()
-        checkOpenSections(['topics'])
-        measureMenu(true)
-    })
-    it('Each open menu section is entirely unfolded and you can scroll the entire menu', () => {
-        init(30, 2)
-        checkOpenSections(['topics', 'activeLayers'])
-        measureMenu(true)
-        checkScrollbarVisibility(false, false)
+            cy.get(menuTopicHeaderSelector).click()
+            waitForAnimationsToFinish()
+            checkOpenSections(['topics'])
+            measureMenu(true)
+        })
+        it('Each open menu section is entirely unfolded and you can scroll the entire menu', () => {
+            init(30, 2)
+            checkOpenSections(['topics', 'activeLayers'])
+            measureMenu(true)
+            checkScrollbarVisibility(false, false)
 
-        cy.get('[data-cy="menu-tray"]').then(($body) => {
-            cy.wrap($body).invoke('outerHeight').should('eq', 498)
-            cy.wrap($body).invoke('prop', 'scrollHeight').should('eq', 1571)
+            cy.get('[data-cy="menu-tray"]').then(($body) => {
+                cy.wrap($body).invoke('outerHeight').should('eq', 498)
+                cy.wrap($body).invoke('prop', 'scrollHeight').should('eq', 1571)
+            })
+        })
+        it('no scrolling if menus are small enough', () => {
+            init(3, 2)
+            checkOpenSections(['topics', 'activeLayers'])
+            measureMenu(false)
+            checkScrollbarVisibility(false, false)
         })
     })
-    it('no scrolling if menus are small enough', () => {
-        init(3, 2)
-        checkOpenSections(['topics', 'activeLayers'])
-        measureMenu(false)
-        checkScrollbarVisibility(false, false)
+    context('Landscape mobile', () => {
+        beforeEach(() => {
+            cy.viewport(568, 320) // Landscape mobile for the default viewport (320 x 568)
```

Comment:
if I had one thing to say about this code, it's that I don't like hard coded values within the code (I would rather have a constant files for cypress tests). However, this is how it is done in every single test with a different viewport, so I'll suggest we keep it as it is, and I'll create a ticket to gather constants (and functions) in a single file.

Reply:
Yes, I completely agree with you. It's better to have it as a constant, like LANDSCAPE_MOBILE, PORTRAIT_MOBILE, etc. Let's wait for a new ticket :)